src/lib/Containers/Inbox.tsx

```diff
@@ -34,14 +36,14 @@ export class Inbox extends React.Component<Props, State> {
     }
 
     this.setState({ fetchingData: true })
-    this.props.relay.refetch(
-      {},
-      null,
-      () => {
-        this.setState({ fetchingData: false })
-      },
-      { force: true }
-    )
+
+    // Force-fetch self; this will update Active Bids but not Conversations (there will be a warning)
+    this.props.relay.refetch({}, {}, null, { force: true })
+
+    // Allow Conversations to properly force-fetch itself
+    this.conversations.refs.component.refreshConversations(() => {
```

Comment:
Where does the `refs.component` part come from?

Reply:
It seems to be the only way to get to the actual `Conversations` component because it's wrapped in the `PaginationContainer`. This was kind of difficult to figure out because we used to be able to access child methods from a ref directly.

Reply:
This is what `this.conversations` looks like:
<img width="783" alt="screen shot 2017-12-04 at 2 35 17 pm" src="https://user-images.githubusercontent.com/2712962/33555347-521d6252-d900-11e7-986a-493d8384743f.png">

Reply:
Ah right, the first component ref is the Relay container and then from there we get our actual component. Can you add a comment that explains that?